src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,20 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import java.util.Random;
+
 public class HelloWorld {
```

Comment:
Task: _In the main() method we need to create an array of figures (the size of array can be 3 or 6, it doesn't matter). The first half of figures in this array should be generated with random parameters._
I can not see this array with two halfs. First half in the array, second half only drawed.